mteb/benchmarks/benchmarks.py

```diff
@@ -308,6 +308,24 @@ def load_results(
     citation=None,
 )
 
+MTEB_RETRIEVAL_MEDICAL = Benchmark(
+    name="MTEB(Medical)",
+    tasks=get_tasks(
+        tasks=[
+            "CUREv1",
+            "NFCorpus",
+            "SCIDOCS",
+            "TRECCOVID",
+            "SciFact",
+            "MedicalQARetrieval",
+            "PublicHealthQA",
+        ]
+    ),
+    description="Medical benchmarks from MTEB",
```

Comment:
Maybe worth adding a selection procedure / curation rationale here. 

It seems like you haven't included e.g. MedrxivClusteringS2SFast, MedrxivClusteringP2PFast

It also seems to be somewhat multilingual, but does not include:

- CMedQAv1

There are also some datasets that do include medical, e.g. AutoRAGRetrieval, but do not target the medical domain per say. I also think these are fine to ignore.

You have also included e.g. scidocs even though it is not annotated for "medical"?

Reply:
It might also be worth differentiating between medical within a healthcare system (search on patient journals) and within e.g. medical journals or QA as these might be quite different. I would make it clear in the description as well.

I know other descriptions are quite small, but we can reasonably add a paragraph in the new leaderboard.

Reply:
Great catch! We were too focused on strictly adding retrieval tasks and overlooked the clustering tasks. Thanks for flagging them @KennethEnevoldsen.

Here’s what we’re proposing:
	•	SCIDOCS:  Removed. While it’s an interesting dataset on scientific literature, we agree it’s too broad for inclusion in this specific benchmark.
	•	TRECCOVID-PL and SciFact-PL: Added. These fit the multilingual focus, and their English counterparts are already included. We also added CMedQAv2 and CmedqaRetrieval.
	•	MedrxivClusteringP2P and MedrxivClusteringS2S: Added as proposed, as they fit well with the type of content we’re targeting.
	•	VieMedEVBitextMining: Not included. Despite being medically oriented, it is a translation mining task and doesn’t quite fit the medical retrieval context.

Regarding the benchmark scope, while I agree the medical field is quite broad, all the current medical tasks in MTEB fall under the medical information retrieval category, rather than EMR search or its derivatives. My concern is that narrowing the scope too much could limit the benchmark’s growth potential, especially if we want to include tasks related to EMR search or patient education in the future since creating a second benchmark for those tasks would add unnecessary fragmentation. Of course, if the benchmark eventually grows to include a large number of tasks covering diverse use cases in the medical field, we can split it into more specific benchmarks down the line. I've clarified the description for now !

Also, if you agree with this line of thought, I have renamed the benchmark to `medMTEB`, since it sounds more catchy haha

If you agree, please go ahead and merge this PR. I’ll run the models on the newly added tasks tomorrow and open a PR with all the results so they’re ready to be displayed on the leaderboard 💪

Reply:
Thanks! I'm happy to wait for a second opinion, but I'm not strongly attached to the `medMTEB` naming either, it was just a suggestion since it sounded nice. So if that's the only thing left blocking this PR I don't have a problem changing it back.

Reply:
thanks for the great PR! re. naming, I agree that following the naming convention might be better; also easier for users that want to evaluate on it to grab tasks in this benchmark with the correct name, since we have other domain specific ones like MTEB(law) etc.

Reply:
Thanks for chiming in! I've renamed it back to `MTEB(Medical)`